docs/getting-started/introduction.md

```diff
@@ -27,6 +27,8 @@ provide a full-fledged solution for composing micro frontends:
   React][placing-feature-apps-on-a-web-page-using-react], but also allows the
   integration of micro frontends that are [built with any other frontend
   technology][dom-feature-app] (e.g. Vue.js, Angular, Web Components).
+- [`@feature-hub/dom`][dom-api] – A library for building a Feature Hub
```

Comment:
Maybe we shouldn't advertise the package here yet. The documentation has not yet been adapted and the sentence is rather unfinished.